### PR TITLE
BuildScriptFor_jboss-marshalling

### PR DESCRIPTION
**Pull request by santosh653**
[INFO]
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ jboss-marshalling-osgi ---
[INFO] Installing /home/ubuntu/jboss-marshalling/jboss-marshalling/osgi/target/jboss-marshalling-osgi-2.0.12.Final.jar to /home/ubuntu/.m2/repository/org/jboss/marshalling/jboss-marshalling-osgi/2.0.12.Final/jboss-marshalling-osgi-2.0.12.Final.jar
[INFO] Installing /home/ubuntu/jboss-marshalling/jboss-marshalling/osgi/dependency-reduced-pom.xml to /home/ubuntu/.m2/repository/org/jboss/marshalling/jboss-marshalling-osgi/2.0.12.Final/jboss-marshalling-osgi-2.0.12.Final.pom
[INFO] Installing /home/ubuntu/jboss-marshalling/jboss-marshalling/osgi/target/jboss-marshalling-osgi-2.0.12.Final-sources.jar to /home/ubuntu/.m2/repository/org/jboss/marshalling/jboss-marshalling-osgi/2.0.12.Final/jboss-marshalling-osgi-2.0.12.Final-sources.jar
[INFO]
[INFO] --- maven-bundle-plugin:2.4.0:install (default-install) @ jboss-marshalling-osgi ---
[INFO] Writing OBR metadata
[INFO] Installing org/jboss/marshalling/jboss-marshalling-osgi/2.0.12.Final/jboss-marshalling-osgi-2.0.12.Final.jar
[INFO] Installing org/jboss/marshalling/jboss-marshalling-osgi/2.0.12.Final/jboss-marshalling-osgi-2.0.12.Final-sources.jar
[INFO] Writing OBR metadata
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for JBoss Marshalling Parent 2.0.12.Final:
[INFO]
[INFO] JBoss Marshalling Parent ........................... SUCCESS [  8.468 s]
[INFO] JBoss Marshalling API .............................. SUCCESS [ 22.004 s]
[INFO] JBoss Marshalling River ............................ SUCCESS [  1.697 s]
[INFO] JBoss Marshalling Serial ........................... SUCCESS [  0.429 s]
[INFO] JBoss Marshalling Tests ............................ SUCCESS [ 17.895 s]
[INFO] JBoss Marshalling OSGi Bundle ...................... SUCCESS [  3.269 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  54.419 s
[INFO] Finished at: 2021-06-04T15:36:38Z
[INFO] ------------------------------------------------------------------------
ubuntu@santoshkulkarni:~/jboss-marshalling$

### CODE REVIEW

File: j/jboss-marshalling/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+Apache License
```
Comment (gururajrkatti) [inline j/jboss-marshalling/LICENSE:1]: Indentation issue

File: j/jboss-marshalling/jboss-marshalling_ubuntu_20.04.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# ----------------------------------------------------------------------------
+#
+# Package           : jboss-marshalling
+# Version           : 0.1.0
```
Comment (gururajrkatti) [inline j/jboss-marshalling/jboss-marshalling_ubuntu_20.04.sh:4]: Kindly update the version

**Review by gururajrkatti — CHANGES_REQUESTED** [review]

**Comment by gururajrkatti** [comment]
As per the sheet, it needs to build river component. However in the logs above I see osgi being built. Pl. ensure the required jar is being built

**Comment by santosh653** [comment]
@gururajrkatti , Hi Guru The Build is successfully building River Component & the Jar: 2.0.9.Final.jar as per the Sheet.,
Thanks.,
===========================================
 --- maven-jar-plugin:3.1.0:jar (default-jar) @ tests ---
[INFO] Building jar: /home/ubuntu/jboss-marshalling/jboss-marshalling/tests/target/tests-2.0.9.Final.jar
[INFO]
[INFO] --- maven-source-plugin:3.0.1:jar-no-fork (attach-sources) @ tests ---
[INFO] Building jar: /home/ubuntu/jboss-marshalling/jboss-marshalling/tests/target/tests-2.0.9.Final-sources.jar
[INFO]
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ tests ---
[INFO] Installing /home/ubuntu/jboss-marshalling/jboss-marshalling/tests/target/tests-2.0.9.Final.jar to /home/ubuntu/.m2/repository/org/jboss/marshalling/tests/2.0.9.Final/tests-2.0.9.Final.jar
[INFO] Installing /home/ubuntu/jboss-marshalling/jboss-marshalling/tests/pom.xml to /home/ubuntu/.m2/repository/org/jboss/marshalling/tests/2.0.9.Final/tests-2.0.9.Final.pom
[INFO] Installing /home/ubuntu/jboss-marshalling/jboss-marshalling/tests/target/tests-2.0.9.Final-sources.jar to /home/ubuntu/.m2/repository/org/jboss/marshalling/tests/2.0.9.Final/tests-2.0.9.Final-sources.jar


[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ jboss-marshalling-osgi ---
[INFO] Installing /home/ubuntu/jboss-marshalling/jboss-marshalling/osgi/target/jboss-marshalling-osgi-2.0.9.Final.jar to /home/ubuntu/.m2/repository/org/jboss/marshalling/jboss-marshalling-osgi/2.0.9.Final/jboss-marshalling-osgi-2.0.9.Final.jar
[INFO] Installing /home/ubuntu/jboss-marshalling/jboss-marshalling/osgi/dependency-reduced-pom.xml to /home/ubuntu/.m2/repository/org/jboss/marshalling/jboss-marshalling-osgi/2.0.9.Final/jboss-marshalling-osgi-2.0.9.Final.pom
[INFO] Installing /home/ubuntu/jboss-marshalling/jboss-marshalling/osgi/target/jboss-marshalling-osgi-2.0.9.Final-sources.jar to /home/ubuntu/.m2/repository/org/jboss/marshalling/jboss-marshalling-osgi/2.0.9.Final/jboss-marshalling-osgi-2.0.9.Final-sources.jar
[INFO]
[INFO] --- maven-bundle-plugin:2.4.0:install (default-install) @ jboss-marshalling-osgi ---
[INFO] Installing org/jboss/marshalling/jboss-marshalling-osgi/2.0.9.Final/jboss-marshalling-osgi-2.0.9.Final.jar
[INFO] Installing org/jboss/marshalling/jboss-marshalling-osgi/2.0.9.Final/jboss-marshalling-osgi-2.0.9.Final-sources.jar
[INFO] Writing OBR metadata
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for JBoss Marshalling Parent 2.0.9.Final:
[INFO]
[INFO] JBoss Marshalling Parent ........................... SUCCESS [  0.923 s]
[INFO] JBoss Marshalling API .............................. SUCCESS [  4.864 s]
[INFO] JBoss Marshalling River ............................ SUCCESS [  1.303 s]
[INFO] JBoss Marshalling Serial ........................... SUCCESS [  0.391 s]
[INFO] JBoss Marshalling Tests ............................ SUCCESS [ 14.735 s]
[INFO] JBoss Marshalling OSGi Bundle ...................... SUCCESS [  1.426 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  24.287 s
[INFO] Finished at: 2021-06-09T06:21:30Z
[INFO] ------------------------------------------------------------------------
ubuntu@santoshkulkarni:~/jboss-marshalling$ ./jboss-marshalling_ubuntu_20.04.sh 2.0.9.Final

File: j/jboss-marshalling/jboss-marshalling_ubuntu_20.04.sh [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+
+export REPO=https://github.com/jboss-remoting/jboss-marshalling
+
+if [ -z "$1" ]; then
+  export VERSION="2.0.12.Final"
```
Comment (gururajrkatti) [inline j/jboss-marshalling/jboss-marshalling_ubuntu_20.04.sh:23]: Is it 2.0.2 or 2.0.12 ?

**Review by gururajrkatti — CHANGES_REQUESTED** [review]

**Comment by santosh653** [comment]
@gururajrkatti I have updated to correct version 2.0.12. Thanks.,

**Review by gururajrkatti — APPROVED** [review]
ok

**Pull request merged**